Makefile.variables

```diff
@@ -17,4 +17,4 @@
 VERSION?=$(shell git describe --always --long --tags || echo 0.0.1)
 IMAGE_SERVER?=europe-west3-docker.pkg.dev
 IMAGE_REGISTRY?=$(IMAGE_SERVER)/fdc-public-docker-registry/kuberpult
-KUBERPULT_BUILDER?=europe-west3-docker.pkg.dev/fdc-public-docker-registry/kuberpult/kuberpult-builder:0.0.1
+KUBERPULT_BUILDER?=europe-west3-docker.pkg.dev/fdc-public-docker-registry/kuberpult/kuberpult-builder:0.4.21-14-g146b8ea
```

Comment:
Are we setting this manually?

Reply:
Yes, this will always be provided by humans.